src/screens/LogIn.tsx

```diff
@@ -38,6 +40,37 @@ export default class LogIn extends Component<ILogInProps, ILogInState> {
       : true;
   };
 
+  onEmailLogIn = async (formikProps: ILogInFormikProps) => {
+    try {
+      await auth().signInWithEmailAndPassword(formikProps.values.email, formikProps.values.password);
+      formikProps.handleSubmit();
+      this.props.navigation.navigate('LoggedTab');
+    } catch (error) {
+      if (error instanceof Error) {
+        Alert.alert('Error', error.message);
+      } else {
+        console.error(error);
```

Comment:
Don't leave code like console or other debugger on GitHub.

Reply:
@YilKanda, I pushed a new commit removing the `console.error`s.
Is it necessary to remove the `Aler.alert`s?, because we are using it for displaying to the user pop up errors like "Invalid password or user" provided by Firebase.